model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -82,32 +83,37 @@ data class PackageConfiguration(
         }
     }
 
-    fun matches(otherId: Identifier, provenance: Provenance): Boolean {
-        @Suppress("ComplexCondition")
-        if (!id.type.equals(otherId.type, ignoreCase = true) ||
-            id.namespace != otherId.namespace ||
-            id.name != otherId.name ||
-            id.version != otherId.version
-        ) {
-            return false
+    private fun Identifier.matches(otherId: Identifier, supportVersionRange: Boolean): Boolean {
+        val basePropertiesMatch = id.type.equals(
+            otherId.type, ignoreCase = true
+        ) && id.namespace == otherId.namespace && id.name == otherId.name
```

Comment:
Would it be possible to re-use the code from `PackageCuration.isApplicableDisregardingVersion()` for matching the `identifier` ? e.g. use the exact same code?